pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -88,21 +85,265 @@ class InvalidSelectionError(DNFManagerError):
     """The software selection couldn't be resolved."""
 
 
+class DNFConfigWrapper(object):
+    """This is a wrapper of a DNF config object."""
+
+    def __init__(self, config):
+        """Wrap the DNF config object."""
+        self._config = config
+
+    @property
+    def baseurl(self):
+        return self._config.get_baseurl_option().get_value()
```

Comment:
This seems not necessary as dnf5 is already providing these attributes in Python bindings. Check [the implementation](https://github.com/rpm-software-management/dnf5/blob/5.2.6.2/bindings/libdnf5/conf.i#L118) in our SWIG interface file and e.g. [this test](https://github.com/rpm-software-management/dnf5/blob/5.2.6.2/test/python3/libdnf5/conf/test_option.py#L59).

Reply:
Oh, wow, how did I miss that? It was even done in the same PR as the renaming, but I guess I focused too much on how the wrapped had been done before that I didn't notice. This is really neat, I will change the PR to drop the wrapper and use the attributes directly.

Reply:
I think this is the case, when presenting a simple demo of new features added to our projects during the sprint review or retrospective would be useful. :slightly_smiling_face: